src/components/authentication/Login.tsx

```diff
@@ -5,20 +5,31 @@ import Input from "../UI/Input";
 import { MdLockOutline } from "react-icons/md";
 import { BsGoogle } from "react-icons/bs";
 import Card from "../UI/Card";
-import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+import axios from "axios";
 
+const url = "/users/login";
 const Login: FC = () => {
-  const [email, setEmail] = useState<string>("");
+  const [username, setUsername] = useState<string>("");
   const [password, setPassword] = useState<string>("");
-  let navigate = useNavigate();
 
   const handleInputPassword = (e: ChangeEvent<HTMLInputElement>) => {
     setPassword(e.target.value);
   };
 
-  const handleSubmitForm = (e: SyntheticEvent) => {
+  const handleSubmitForm = async (e: SyntheticEvent) => {
     e.preventDefault();
-    setEmail("");
+    try {
+      const response = await axios.post(url, { username, password });
+      console.log(response.data);
```

Comment:
save the `response` in state or somewhere not in console

Reply:
Noted, i have saved in useState hook
`setData(response.data.access_token);`